src/sass/utilities/_utilities-color.scss

```diff
@@ -3,28 +3,17 @@
 
 /* stylelint-disable */
 
-@each $key, $value in $color-map {
```

Comment:
One more thing I forgot. You'll need to update the copyright year here to be 2019 since this is a new piece of code.

Reply:
This file was around before these updates, previously using `$colors-map` to loop and produce background/foreground colors.